orttraining/orttraining/models/bert/main.cc

```diff
@@ -781,6 +784,8 @@ int main(int argc, char* argv[]) {
   BertParameters params;
   OrtParameters ort_params{};
   RETURN_IF_FAIL(ParseArguments(argc, argv, params, ort_params));
+  bool keep_looping = params.debug_break;
+  while(keep_looping);
```

Comment:
(nit):  remove debugging code :)

Reply:
This is a breakthrough feature :) Instead of inserting temporary debug code this breaks when the flag is switched on.

---
In reply to: [505116872](https://github.com/microsoft/onnxruntime/pull/4443#discussion_r505116872) [](ancestors = 505116872)